pulseapi/settings.py

```diff
@@ -53,6 +51,7 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'corsheaders',
+    'storages',
```

Comment:
Trying to spin up my instance, this can't be found. `requirements.txt` updates needed?